app/src/main/java/com/globant/tutoria/subscriberssearch/fragment/SubscriberListFragment.java

```diff
@@ -31,8 +36,22 @@
     protected RecyclerView recyclerViewSubscribers;
 
     @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        switch (item.getItemId()) {
+            case R.id.menu_toggle_log:
+                Log.e("menu", "CLICK");
+                return true;
+            default:
+                return super.onOptionsItemSelected(item);
+        }
+    }
+
+    ProgressDialog progressDialog;
```

Comment:
Please put the access modifier and put all the class fields at the top of the class, just bellow the class name.